std/file.d

```diff
@@ -5105,8 +5105,15 @@ string tempDir() @trusted
 ///
 @safe unittest
 {
+    import std.ascii : letters;
+    import std.conv : to;
     import std.path : buildPath;
-    auto myFile = tempDir.buildPath("my_tmp_file");
```

Comment:
Why not `auto myFile = deleteme ~ "my_tmp_file";`?

Seems like this is more complicated than it needs to be.

Reply:
Because this is the public example for `tempDir` - wouldn't make sense to not use it.

Reply:
`auto myFile = tempDir.buildPath(deleteme);`?

Reply:
`deleteme = "/tmp/<randomid>.<pid>`